### DISCUSSION TITLE
How to document default error models for all other status codes than those mentioned separately

### DISCUSSION THREAD

**Discussion by bhanuprakash-1** [fastapi/fastapi#6074] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

I want something like the below, where I can set a response model for all the other http status codes
```python
@app.get("/items/{item_id}", responses={200: {"model": mySuccessResponseModel}, Any: {"model": myErrorResponseModel}})
async def read_item(item_id: str):

```

### Description

I want to set a single response model for all other status codes other than those mentioned separatly. The reason why I want is, so that my swagger doc will have the below format. 

      responses:
        '200':
          description: Success
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/mySuccessResponseModel'
        default:
          description: Error
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/myErrorResponseModel'

I have this requirement, because if my swagger is like above, the clients generated with AutoRest client generator, will throw an Rest exception in c# with parameters same as the model defaultErrorResponse.

### Operating System

Linux

### FastAPI Version

0.65.2

### Python Version

3.6.9

**Comment by notypecheck** [comment, marked as answer]
OpenAPI spec allows you to specify response for a range of status codes:

https://swagger.io/docs/specification/describing-responses/
```py
@app.get(
    path="/",
    responses={
        200: {"model": None},
        "5XX": {"model": int},
    }
)
async def index() -> None:
    return None
```
Also update your python 😳 3.11 is out and it's free
